packages/region-config-resolver/package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "@aws-sdk/region-config-resolver",
+  "version": "1.0.0",
```

Comment:
Contrary to my previous comment, this probably should be `"3.0.0"` after confirming with the JS SDK team.